ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestCase.java

```diff
@@ -295,4 +318,178 @@ private static String standardizeTopology(final String topology) {
     return result.toString();
   }
 
+  void createInputTopics(final FakeKafkaService fakeKafkaService) {
```

Comment:
yeah, I put them here since the topics collection was not exposed in TestCase. I think its better to move them to `TestExecutor` as you suggested. Made the change.